bundles/org.openhab.binding.govee/src/main/java/org/openhab/binding/govee/internal/GoveeHandler.java

```diff
@@ -318,12 +317,16 @@ public void updateDeviceState(@Nullable StatusResponse message) {
         if (newColorTempInKelvin != lastColorTempInKelvin) {
             logger.trace("Color-Temperature Status: old: {} K {}% vs new: {} K", lastColorTempInKelvin,
                     newColorTempInPercent, newColorTempInKelvin);
-            updateState(CHANNEL_COLOR_TEMPERATURE_ABS, new QuantityType<>(lastColorTempInKelvin, Units.KELVIN));
-            updateState(CHANNEL_COLOR_TEMPERATURE, new PercentType(newColorTempInPercent));
-        }
+            if (lastColorTempInKelvin >= 2000 && lastColorTempInKelvin <= 9000) {
```

Comment:
Minor, but perhaps these values could be extracted to constants in `GoveeBindingConstants` for better visibility, since they need to be kept in sync with the same values provided in the thing type declaration.

And then I checked: They are already there: `COLOR_TEMPERATURE_MIN_VALUE` and `COLOR_TEMPERATURE_MAX_VALUE`.

Reply:
That's a lot better than keeping the old state in my book. In this specific case, which looks like rounding or off-by-one issues to me, I'd prefer clamping though (as mentioned), but setting to UNDEF would be OK to me as well.

Reply:
And, just to be extra clear: this check still uses the wrong variable (old state, not new state).

Reply:
I personally agree that rounding to the valid range would probably be preferable (assuming that a value like 9001 is not completely random, but rather meant at close to 9000), but it was rejected previously. As next choice `UNDEF` would be fine too, and just completely ignoring and keeping old state is probably the least desirable.

Reply:
@lsiepel - that probably means that the current test is not effective?

Reply:
Ok. Thought about it and changed the logic.

I really don't like interpreting/clamping invalid values. But ok. This binding already corrected the low side of the range. So i adjusted it and now it also corrects it at the upper side.